operator/pkg/clusterconfiguration/cel_patcher.go

```diff
@@ -19,14 +19,28 @@ import (
 type Template[T any] struct {
 	Content T `json:"content,omitempty" yaml:"content,omitempty"`
 
-	Fixups []Fixup `json:"fixups,omitempty"`
+	Fixups   []Fixup `json:"fixups,omitempty"`
+	Warnings []error
```

Comment:
nit: should this be colocated with `Content` to better indicate that both `Content` and `Warnings` will be mutated after calling `Fixup`?

Reply:
Yeah.